_faqs/02-06-what-is-fluentbit-and-fluentd.markdown

```diff
@@ -0,0 +1,10 @@
+---
+question: What is Fluentd and Fluent Bit?
+category: Plugins
+---
+
+[Fluentd](https://calyptia.com/download/) is a data collector for log data collection, processing, and forwarding. It’s written in Ruby and supports over 500 plugins including data sources, data output, parsers, formatters, and filters.
+
+[Fluent Bit](https://calyptia.com/download/) is an agent that collects data from different sources, enriches them with filters, and sends them to multiple destinations. It’s designed with performance in mind, meaning it is optimized for high throughput and low CPU and memory usage. It’s written in C and has an architecture that supports more than 70 plugins for inputs, filters, and outputs.
+
+Both Fluentd and Fluent Bit have OpenSearch plugins.
```

Comment:
This is confusing. It sounds like they have OpenSearch plugins (like alerting, ISM, etc.) not plugins that enable OpenSearch on Fluentd and Fluent Bit.

Reply:
I'll remove that sentence for now and loop back around when we have better instructions on implementing Fluent unto OpenSearch